sbi/utils/sbiutils.py

```diff
@@ -443,6 +627,165 @@ def batched_mixture_mv(matrix: Tensor, vector: Tensor) -> Tensor:
     return torch.einsum("bcij,bcj -> bci", matrix, vector)
 
 
+def optimize_potential_fn(
+    potential_fn: Callable,
+    inits: Tensor,
+    dist_specifying_bounds: Optional[Any] = None,
+    num_iter: int = 1_000,
+    num_to_optimize: int = 100,
+    learning_rate: float = 0.01,
+    save_best_every: int = 10,
+    show_progress_bars: bool = False,
+    interruption_note: str = "",
+) -> Tuple[Tensor, Tensor]:
+    """
+    Returns the location of the maximum of a `potential_fn`.
```

Comment:
what does location mean here? the corresponding parameter?

Reply:
yes! I changed the docstring